server/passwordReset/queries.ts

```diff
@@ -78,7 +78,7 @@ export const updatePasswordReset = (
 			}
 
 			/* Promisify the setPassword function, and use .update to match API convention */
-			const setPassword = promisify((userData as any).setPassword.bind(userData));
```

Comment:
there are a few more of these strewn around, but they require some more effort to fix, should be done after other sub-prs are merged